### ISSUE TITLE
Contact Zulko

### ISSUE THREAD

**Issue by avemeva** [Edinburgh-Genome-Foundry/DnaWeaver#2]
Hey @Zulko, what is the best way to contact u if I have questions regarding the DnaWeaver functionality?

**Comment by Zulko** [comment]
Hey thanks for your interest in Weaver! In case it helps there are introductory slides [here](https://github.com/Edinburgh-Genome-Foundry/egf-shared-documents/tree/master/slideshows/dnaweaver_presentation_iwbda_2019) and a  (never completely finished) paper [there](https://github.com/Zulko/dna_weaver_paper/blob/master/DNA_Weaver.pdf).

For general questions about capabilities the best would be to open a Github issue so it's transparent and easily findable by others in the future. If there are details you would prefer not to share publicly, I think the usually preferred method would be to ask @veghp (who is the current EGF software lead) via email at `egf-software@ed.ac.uk`, although for Weaver in particular I probably have the most knowledge, and you can find my personal email address in the git repo:

```
git clone git@github.com:Edinburgh-Genome-Foundry/DnaWeaver.git
cd DnaWeaver
git show HEAD~7  # my email at the top
```

**Comment by avemeva** [comment]
Thank u! I will check out the docs first, and if still have questions put it here.

**Comment by pvegh** [comment]
Hi @Zulko, you mixed me up with @veghp :)

**Comment by Zulko** [comment]
Sorry for the noise @pvegh ! I didn't write this in the Github so didn't have the autocompletion check.

**Comment by avemeva** [comment]
@Zulko I really-really like your presentation! Is there a video by any chance? The sequence decomposition is brilliant.

**Comment by Zulko** [comment]
Thanks! I am really happy about the methods I used in Weaver (and the fact that the A* algorithm works so well is really cool). Unfortunately I only gave this presentation once and I don't think it was recorded. I have a plan to write a "transcript" of it as some kind of very long blog post in the future, but no time frame for that. For now, the closest I have is the abstract ([page 17 here](https://www.iwbdaconf.org/2019/docs/IWBDA19Proceedings.pdf)) and the manuscript I linked above.
